code/game/objects/items/food/misc.dm

```diff
@@ -1,39 +1,49 @@
 
 ////////////////////////////////////////////OTHER////////////////////////////////////////////
 
-/obj/item/food/cheesewheel
+
+/obj/item/food/cheese/wheel
 	name = "cheese wheel"
 	desc = "A big wheel of delcious Cheddar."
 	icon_state = "cheesewheel"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 10, /datum/reagent/consumable/nutriment/protein = 5, /datum/reagent/consumable/nutriment/vitamin = 5) //Hard cheeses contain about 25% protein
 	w_class = WEIGHT_CLASS_NORMAL
-	tastes = list("cheese" = 1)
-	foodtypes = DAIRY
+	rat_heal = 35
 
-/obj/item/food/cheesewheel/Initialize()
+/obj/item/food/cheese/wheel/Initialize()
 	. = ..()
 	AddComponent(/datum/component/food_storage)
 
-/obj/item/food/cheesewheel/MakeProcessable()
-	AddElement(/datum/element/processable, TOOL_KNIFE, /obj/item/food/cheesewedge, 5, 30)
+/obj/item/food/cheese/wheel/MakeProcessable()
+	AddElement(/datum/element/processable, TOOL_KNIFE, /obj/item/food/cheese, 5, 30)
 
-/obj/item/food/royalcheese
+/obj/item/food/cheese/royal
 	name = "royal cheese"
 	desc = "Ascend the throne. Consume the wheel. Feel the POWER."
 	icon_state = "royalcheese"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 15, /datum/reagent/consumable/nutriment/vitamin = 5, /datum/reagent/gold = 20, /datum/reagent/toxin/mutagen = 5)
 	w_class = WEIGHT_CLASS_BULKY
-	tastes = list("cheese" = 4, "royalty" = 1)
-	foodtypes = DAIRY
+	tastes = list("cheese" = 4, "royalty" = 1)	
+	rat_heal = 70
 
-/obj/item/food/cheesewedge
+/obj/item/food/cheese
 	name = "cheese wedge"
 	desc = "A wedge of delicious Cheddar. The cheese wheel it was cut from can't have gone far."
 	icon_state = "cheesewedge"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 2, /datum/reagent/consumable/nutriment/protein = 1, /datum/reagent/consumable/nutriment/vitamin = 1)
 	tastes = list("cheese" = 1)
 	foodtypes = DAIRY
 	w_class = WEIGHT_CLASS_SMALL
+	var/rat_heal = 10
```

Comment:
Document this with a leading `/// Bla bla bla` comment--https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md